www/lobby/index.ts

```diff
@@ -0,0 +1,60 @@
+import { Router, Request, Response } from 'express';
+import { Lobby } from '../../lib/lobby';
+import { User } from '../../lib/user';
+import { validateJwt } from '../../utils/jwt';
+import { generateLobbyId } from '../../utils/lobby';
+import { lobby_id_router } from './:id';
+
+export const lobby_router = Router();
+
+lobby_router.use('/midtest', validateJwt);
+
+// Creates a new lobby and returns the lobby id w/ lobby data
+/**
+ * Body Params: lobbyName, theme, id, refreshToken
+ */
+lobby_router.post('/', async (req: Request, res: Response) => {
+  const lobbyName = req.body.lobbyName;
+  const theme = req.body.theme;
+  const userId = req.body.id;
+  const participants = [userId];
+  const songIds : string[] = [];
+
+  const manager = await User.fromId(userId) ?? null;
+  if (!manager) {
+    res.status(404).json('user dne');
+    return;
+  }
+
+  let retryNum = 0;
+  let newLobbyId = generateLobbyId(userId, retryNum);
+  let exists = await Lobby.fromId(newLobbyId);
+  while (exists) {
+    retryNum++;
+    newLobbyId = generateLobbyId(userId, retryNum);
+    exists = await Lobby.fromId(newLobbyId);
+  }
+
+  const lobby = new Lobby(newLobbyId, {
+    theme: theme,
+    name: lobbyName,
+    managerId: userId,
+    participants: participants,
+    spotifyPlaylistId: undefined,
```

Comment:
u dont need to define this! if its not defined then it will be set to `undefined`